libs/lexical_cast/examples/variant_to_long_double.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright Antony Polukhin, 2013-2019.
```

Comment:
I think we should remove this particular test as it introduces a dependency on Boost variant.